### ISSUE TITLE
Question about vue-router and aria-current

### ISSUE THREAD

**Issue by darrinmn9** [accessible-app/vuejs#5]
with the recent release of vue-router `v3.2.0` https://github.com/vuejs/vue-router/releases/tag/v3.2.0, would the `setAriaCurrent` method in `App.vue` no longer be necessary?

**Comment by roybarber** [comment]
@darrinmn9 Correct, as of 3.2.0 the ariaCurrent function is no longer required, vue automatically handles this now.
